README.md

```diff
@@ -1,41 +1,62 @@
-# VDOM
+# [VDOM](https://github.com/nteract/vdom)
 
-Write Declarative Pythonic layouts, render in Jupyter frontends, or serialize for rehydration in your web app.
+## Why use VDOM?
 
-:warning: This library is a work in progress. We'd love your input on the exposed API :warning:
+- Write **Declarative** Pythonic layouts.
+- Render the layout in Jupyter and nteract frontends.
+- Serialize layout for rehydration in your web app.
+
+:warning: This library is a work in progress. :warning:
+
+## Check out the power of VDOM!
+
+Start with Python code and run it:
 
 ```python
 from vdom import VDOM, h1, p, img, div, b
 
 VDOM(
     div([
-        h1('Now Incredibly Declarative'),
+        h1('Our Incredibly Declarative Example'),
         p(['Can you believe we wrote this ', b('in Python'), '?']),
         img(src="https://media.giphy.com/media/xUPGcguWZHRC2HyBRS/giphy.gif"),
+        p(['What will', b('you') 'create next?']),
     ])
 )
 ```
 
------------------------
+**Voila!** Your layout is served below:
 
 # Now Incredibly Declarative
 
 Can you believe we wrote this **in Python**?
 
 ![](https://media.giphy.com/media/xUPGcguWZHRC2HyBRS/giphy.gif)
 
------------------------
+What will **you** create next?
 
-## Install
+---
 
-```
+## Install the Python package
+
+```bash
 pip install vdom
 ```
 
-## Development
+## Developer install from source code
```

Comment:
I need to keep this in mind in general, since otherwise `Development` can be conflated. Thanks Carol.